src/sentry/api/endpoints/group_tags.py

```diff
@@ -24,17 +24,8 @@ def get(self, request, group):
         else:
             value_limit = 10
 
-        use_snuba = request.GET.get('enable_snuba') == '1'
```

Comment:
Did you verify with james from the production logs (assuming we have query arguments in nginx logs) if anyone is ever passing enable_snuba=0 in the public api ?

Reply:
@JTCunning Can you help look into this? I assume that *everyone* consuming the public API is not passing any value for enable_snuba though, since it's totally undocumented and added quite recently.

Reply:
If someone is not passing a value for "enable_snuba" (presumably everyone except our web app) it's the same as passing enable_snuba=0 though.

Reply:
From james:
```
SELECT *
FROM clicktail.access_log
WHERE (request_uri LIKE '%enable_snuba=%') AND NOT (request_uri LIKE '%enable_snuba=1%') AND (_date > subtractDays(today(), 7))
LIMIT 10

Ok.

0 rows in set. Elapsed: 106.313 sec. Processed 3.83 billion rows, 292.43 GB (36.05 million rows/s., 2.75 GB/s.)
```